app/logging.py

```diff
@@ -43,7 +45,6 @@ def _configure_watchtower_logging_handler():
     aws_region_name = os.getenv("AWS_REGION_NAME", None)
     log_group = os.getenv("AWS_LOG_GROUP", "platform")
```

Comment:
```suggestion
```

This variable is not in the condition few lines below.

Reply:
Function definition
```python
def _configure_watchtower_logging_handler(logger):
```

The same for __configure_contextual_logging_filter_.
```python
def _configure_contextual_logging_filter(logger):
```

```suggestion
    aws_log_group = os.getenv("AWS_LOG_GROUP", "platform")
```